<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Chromatin Landscapes of Human Lung Cells Predict Potentially Functional COPD GWAS Variants
</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	
    <!-- Customized Bootstrap + Font Awesome + Solarized -->
    <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/favicon.png"/>	
			
	<!-- Google Analytics -->	
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', '269018817', 'cbenway.github.io');
		ga('send', 'pageview');

	</script>
			
</head>

<body>
	
	<div id="header">
		<nav class="navbar navbar-default navbar-static-top" role="navigation">	 
			<div class="container">	
				
				<div class="navbar-header">
					<object class="pull-left logo" data="/images/logo-cb-seq.svg"></object>
					<a class="navbar-brand" href="/">Christopher J. Benway, Ph.D.</a>
				</div>
				
				<ul class="nav navbar-nav navbar-right">
                    <!-- 
                    <li>
                    
                    <a href="/blog/">blog</a>
                    </li> -->
					
					<li class="active">
					
					<a href="/papers/">Papers</a></li>
					
					<li>
					
					<a href="/projects/">Projects</a></li>	  
					
					<li>
					
					<a href="/protocols/">Protocols</a></li>	  	  
				</ul>
			</div>
		</nav>	
	</div>
	
	<div class="container">	
		
	<div class="bigspacer"></div>

<div class="row">
	<div class="col-md-12">
		<div class="media">
			
			<img class="pull-left pad-right media-object" src="/images/papers/moll-exon-fig4.png" style="max-width:150px">
			
			<div class="media-body">
				<div class="title media-heading">
					A systematic analysis of protein-altering exonic variants in chronic obstructive pulmonary disease
				</div>
				<p>
				<div class="smallhead">
					Moll M, Jackson VE, Yu B, Grove ML, London SJ, Gharib SA, Bartz TM, Sitlani CM, Dupuis J, O'Connor GT, Xu H, Cassano PA, Patchen BK, Kim WJ, Park J, Kim KH, Han B, Barr RG, Manichaikul A, Nguyen JN, Rich SS, Lahousse L, Terzikhan N, Brusselle G, Sakornsakolpat P, Liu J, Benway CJ, Hall IP, Tobin MD, Wain LV, Silverman EK, Cho MH*, Hobbs BD*.
					<br>

          </div>
					<div class="bigspacer"></div>
                <div class="smallhead">
					Am J Physiol Lung Cell Mol Physiol.
					(2021)
				</div>
			</div>
		</div>
	</div>
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-md-3">
		<div class="bigspacer"></div>
		<div class="glyphbox note">
			
			<div class="smallhead">
				PDF
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-file-text-o fa-fw"></i>
				<a class="off" href="/pdfs/papers/Moll-exon-COPD_AJPL_2021.pdf">Moll-exon-COPD_AJPL_2021_2021.pdf</a>
			</div>
			<div class="bigspacer"></div>
			
			
			
			
			<div class="smallhead">
				DOI
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-external-link fa-fw"></i>
				<a class="off" href="https://doi.org/10.1152/ajplung.00009.2021">10.1152/ajplung.00009.2021</a>
			</div>
			<div class="bigspacer"></div>
			
			
			<div class="smallhead">
				Share
			</div>
 			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-twitter fa-fw"></i> <a class="off" href="http://twitter.com/share?url=http://cbenway.github.io/edit/master/papers/moll_exon/&text=A%20systematic%20analysis%20of%20protein-altering%20exonic%20variants%20in%20chronic%20obstructive%20pulmonary%20disease" target="_blank">tweet</a>
				
			</div>		
			</div>
	</div>
	<div class="col-md-8">
		<div class="post">
			<h1 id="abstract">Abstract</h1>

<p>Genome-wide association studies (GWASs) have identified regions associated with chronic obstructive pulmonary disease (COPD). GWASs of other diseases have shown an approximately 10-fold overrepresentation of nonsynonymous variants, despite limited exonic coverage on genotyping arrays. We hypothesized that a large-scale analysis of coding variants could discover novel genetic associations with COPD, including rare variants with large effect sizes. We performed a meta-analysis of exome arrays from 218,399 controls and 33,851 moderate-to-severe COPD cases. All exome-wide significant associations were present in regions previously identified by GWAS. We did not identify any novel rare coding variants with large effect sizes. Within GWAS regions on chromosomes 5q, 6p, and 15q, four coding variants were conditionally significant (P < 0.00015) when adjusting for lead GWAS single-nucleotide polymorphisms A common gasdermin B (GSDMB) splice variant (rs11078928) previously associated with a decreased risk for asthma was nominally associated with a decreased risk for COPD [minor allele frequency (MAF) = 0.46, P = 1.8e-4]. Two stop variants in coiled-coil Î±-helical rod protein 1 (CCHCR1), a gene involved in regulating cell proliferation, were associated with COPD (both P < 0.0001). The SERPINA1 Z allele was associated with a random-effects odds ratio of 1.43 for COPD (95% confidence interval = 1.17-1.74), though with marked heterogeneity across studies. Overall, COPD-associated exonic variants were identified in genes involved in DNA methylation, cell-matrix interactions, cell proliferation, and cell death. In conclusion, we performed the largest exome array meta-analysis of COPD to date and identified potential functional coding variants. Future studies are needed to identify rarer variants and further define the role of coding variants in COPD pathogenesis.</p>

<p>* corresponding author</p>

		</div>
	</div>
	<div class="col-md-1"></div>
</div>

	
	</div>
		
	<div id="footer"><span style="display:none">foo</span></div>
	    		
	<!-- MathJax support -->
	<script type="text/javascript"
	 src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
	<!-- MathJax config -->
	<script src="/js/mathjax.config.js"></script>	
			
</body>
</html>
